airbyte-integrations/connectors/source-shopify/unit_tests/test_control_rate_limit.py

```diff
@@ -0,0 +1,153 @@
+#
+# MIT License
+#
+# Copyright (c) 2020 Airbyte
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+
+from functools import wraps
+
+import requests
+
+# Define standard timings in seconds
+SLEEP_ON_LOW_LOAD: float = 0.2
+SLEEP_ON_AVG_LOAD: float = 1.5
+SLEEP_ON_HIGH_LOAD: float = 5.0
+SLEEP_ON_UNKNOWN_LOAD: float = 1.0
+
+test_header_name = "X-Shopify-Shop-Api-Call-Limit"
+test_data_field = "some_data_field"
+test_rate_limit_threshold = 0.9
```

Comment:
Use capital letters  for global variables ([pep8](https://www.python.org/dev/peps/pep-0008/#constants))